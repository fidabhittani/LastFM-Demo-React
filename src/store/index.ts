/**
 * Application redux store with configured middlewares and reducers.
 */

import { applyMiddleware, createStore } from "redux";
import { composeWithDevTools } from "redux-devtools-extension";
import reducers from "reducers";
import thunkMiddleware from "redux-thunk";

export const store = createStore(
  reducers,
  composeWithDevTools(applyMiddleware(thunkMiddleware))
);

export const dispatch = store.dispatch;

export default store;
