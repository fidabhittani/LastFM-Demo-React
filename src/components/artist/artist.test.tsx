import React from "react";
import { shallow } from "enzyme";
import { Provider } from "react-redux";
import Artist from "./index";
import store from "store";

const WrappedReduxArtist = () => (
  <Provider store={store}>
    <Artist />
  </Provider>
);

/** Artist TEST */
describe("Artist Root Component Tests", () => {
  it("Artist Component renders correctly", () => {
    const artist = shallow(<WrappedReduxArtist />);
    expect(artist).toMatchSnapshot();
  });
});
