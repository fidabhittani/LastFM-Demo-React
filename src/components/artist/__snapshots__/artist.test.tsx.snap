// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Artist Root Component Tests Artist Component renders correctly 1`] = `ShallowWrapper {}`;
